### PR TITLE
[muxorch] Skip programming ACL for `active-active` ports

### PR DESCRIPTION
**Pull request by lolyu**
**What I did**
Skip adding ACL for `active-active` ports.

**Why I did it**
Please refer to the HLD for more details: https://github.com/sonic-net/SONiC/pull/1180

**How I verified it**

**Details if related**

### CODE REVIEW

File: orchagent/muxorch.cpp [hunk 1]
```diff
@@ -1278,6 +1284,23 @@ bool MuxOrch::handleMuxCfg(const Request& request)
...
+                cable_type = MuxCableType::ACTIVE_STANDBY;
+            }
+            else if (cable_type_str == "active-active")
+            {
+                cable_type = MuxCableType::ACTIVE_ACTIVE;
+            }
+            else
+            {
+                cable_type = MuxCableType::ACTIVE_STANDBY;
+            }
```
[the remark this answers is not in the document]
Comment (lolyu) [reply orchagent/muxorch.cpp]: updated, removed redundant conditions

**Review by Ndancejic — APPROVED** [review]

**Comment by StormLiangMS** [comment]
@lolyu cherry-picked failed for 202211, could you raise new PR for 202211?

**Pull request merged**